rtrlib/lib/vector.c

```diff
@@ -0,0 +1,80 @@
+
+
+#include "vector.h"
+
+int create_vector(struct lrtr_vector **vector_pointer)
+{
+	const size_t DEFAULT_INITIAL_SIZE = 100;
```

Comment:
please use a power of two for the initial size, keeping it small, e.g. 1 or 2. optionally specifying a starting capacity will be helpful, please add.